docs/_templates/layout.html

```diff
@@ -1,5 +1,10 @@
 {% extends "!layout.html" %}
 
+{% block extrahead %}
+  {{ super() }}
+
+  <script defer data-domain="docs.readthedocs.io" src="https://plausible.io/js/script.js"></script>
```

Comment:
Does this also get added to `dev.`?

Reply:
Good point! 👍🏼 -- This should probably use a Sphinx variable.

Reply:
I updated this to use `READTHEDOCS_PROJECT` variable.

We should create a new `dev.readthedocs.io` on Plausible now. Can you do that and give me permissions?